File: DataLayer/Entities/FaceDetection.cs
using System;
using System.Collections.Generic;

namespace DataLayer.Entities
{
    public class FaceDetection : EntityBase
    {
        public Guid Guid { get; set; }
        public string Name { get; set; }
        public int DnnFaces { get; set; }
        public int HaarFaces { get; set; }
        public int? StatusId { get; set; }
        public Status Status { get; set; }
        public IEnumerable<File> Files { get; set; }
    }
}

File: dataLayer/entities/face_detection.py
from sqlalchemy import Column, String, Integer, Date
from sqlalchemy.orm import relationship

from dataLayer.database_connection import Base


class FaceDetection(Base):
    __tablename__ = 'FaceDetection'

    id = Column(Integer, primary_key=True)
    dnnFaces = Column('DnnFaces', Integer)
    haarFaces = Column('HaarFaces', Integer)
    name = Column('Name', String)
    statusId = Column('StatusId', Integer)
    guid = Column('Guid', String)
    files = relationship("File")
